src/ducks/categories/helpers.js

```diff
@@ -144,4 +144,8 @@ export const getGlobalCurrency = categories => {
   return currency
 }
 
+export const isProfessionalExpense = transaction => {
+  return getCategoryId(transaction) === '600140' && transaction.amount < 0
```

Comment:
Nit: Magic string here. A constant would be a bit more readable.

Reply:
See https://github.com/cozy/cozy-banks/pull/1773/commits/ee020f5439454fb0e963e90ad93b0e5b957228f4